crescent/internal/registry.py

```diff
@@ -74,6 +75,20 @@ def hook(self: MetaStruct[CommandCallback, AppCommandMeta]) -> None:
     return meta
 
 
+class ErrorHandler:
+    __slots__ = (
```

Comment:
Missing `typing.Sequence[str]`

Reply:
I'll add. It's worth noting that there's really no reason to the type this; Mypy (and all type checkers for that matter) infer the value if you directly set it. There's also no reason to have a sequence over tuple, because tuple is what it is and all it will ever be.